server/src/main/java/org/opensearch/action/support/nodes/TransportNodesAction.java

```diff
@@ -202,11 +202,16 @@ protected NodeResponse nodeOperation(NodeRequest request, Task task) {
 
     /**
      * resolve node ids to concrete nodes of the incoming request
-     **/
-    protected void resolveRequest(NodesRequest request, ClusterState clusterState) {
+     *
+     * @return
+     */
+    protected DiscoveryNode[] resolveRequest(NodesRequest request, ClusterState clusterState) {
```

Comment:
Is it Ok to introduce this breaking change ? What if plugins extend the class and are consuming this method.